### ISSUE TITLE
[Bug]: Climbers Trail Extract Still Requires RR/Paracord/NoArmour

### ISSUE THREAD

**Issue by HB53** [refringe/OpenExtracts#5]
### Discord Username

harry53

### What happened?

I have 'ignoreCliffRequirements' set to true, but the Climbers Trail on Shoreline still requires the normal Cliff Extract requirements.

With debug turned on I can see that Lighthouse and Reserve both state "Alpinist on MAP has had its paracord, red rebel, and armored rig requirements removed." in the logs. There does not appear to be one for Shoreline though. I've put this in the server log part below.

If you need anything else just @ me on the SPT discord, or send me a PM.

I will also note that I haven't yet had a chance to test the Reserve or Lighthouse cliff extracts.

### SPT Version

3.8.0

### Server Log

```shell
OpenExtracts: Crossroads on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: Smuggler's Boat on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: RUAF Roadblock on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: ZB-1012 on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: ZB-1011 on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: Trailer Park on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: Old Gas Station on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: Dorms V-Ex on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: Dorms V-Ex on Customs has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: EXFIL_ZB013 on Customs has been updated to allow all entry points: Boiler Tanks,Customs.
OpenExtracts: NW Exfil on Interchange has been updated to allow all entry points: MallSE,MallNW.
OpenExtracts: SE Exfil on Interchange has been updated to allow all entry points: MallSE,MallNW.
OpenExtracts: PP Exfil on Interchange has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: Hole Exfill on Interchange has had its backpack requirement removed.
OpenExtracts: Interchange Cooperation on Interchange has been converted to a payment extract.
OpenExtracts: lab_Vent on Laboratory has had its backpack requirement removed.
OpenExtracts: Alpinist_light on Lighthouse has had its paracord, red rebel, and armored rig requirements removed.
OpenExtracts: tunnel_shared on Lighthouse has been converted to a payment extract.
OpenExtracts: Nothern_Checkpoint on Lighthouse has been updated to allow all entry points: Tunnel,North.
OpenExtracts: V-Ex_light on Lighthouse has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: Alpinist on Reserve has had its paracord, red rebel, and armored rig requirements removed.
OpenExtracts: EXFIL_ScavCooperation on Reserve has been converted to a payment extract.
OpenExtracts: EXFIL_vent on Reserve has had its backpack requirement removed.
OpenExtracts: Unity_free_exit on Ground Zero has been updated to allow all entry points: east,west.
OpenExtracts: Sandbox_VExit on Ground Zero has been updated to allow all entry points: east,west.
OpenExtracts: Sandbox_VExit on Ground Zero has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: Sniper_exit on Ground Zero has been updated to allow all entry points: east,west.
OpenExtracts: Scav_coop_exit on Ground Zero has been updated to allow all entry points: east,west.
OpenExtracts: Scav_coop_exit on Ground Zero has been converted to a payment extract.
OpenExtracts: Nakatani_stairs_free_exit on Ground Zero has been updated to allow all entry points: east,west.
OpenExtracts: Tunnel on Shoreline has been updated to allow all entry points: Riverside,Village.
OpenExtracts: Rock Passage on Shoreline has had its chance to be enabled changed from 0% to 50%.
OpenExtracts: CCP Temporary on Shoreline has been updated to allow all entry points: Riverside,Village.
OpenExtracts: Road to Customs on Shoreline has been updated to allow all entry points: Riverside,Village.
OpenExtracts: Road_at_railbridge on Shoreline has been updated to allow all entry points: Riverside,Village.
OpenExtracts: Shorl_V-Ex on Shoreline has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: Smugglers_Trail_coop on Shoreline has been converted to a payment extract.
OpenExtracts: E1 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E2 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E3 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E4 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E5 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E6 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E7_car on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E7_car on Streets of Tarkov has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: E8_yard on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E9_sniper on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: Exit_E10_coop on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: Exit_E10_coop on Streets of Tarkov has been converted to a payment extract.
OpenExtracts: E7 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: E8 on Streets of Tarkov has been updated to allow all entry points: E6_1,E1_2,E2_3,E3_4,E4_5,E5_6.
OpenExtracts: ZB-016 on Woods has been updated to allow all entry points: House,Old Station.
OpenExtracts: Outskirts on Woods has been updated to allow all entry points: House,Old Station.
OpenExtracts: UN Roadblock on Woods has been updated to allow all entry points: House,Old Station.
OpenExtracts: RUAF Gate on Woods has been updated to allow all entry points: House,Old Station.
OpenExtracts: ZB-014 on Woods has been updated to allow all entry points: House,Old Station.
OpenExtracts: South V-Ex on Woods has had its extraction time updated from 60 seconds to 30 seconds.
OpenExtracts: Factory Gate on Woods has been converted to a payment extract.
OpenExtracts: un-sec on Woods has been updated to allow all entry points: House,Old Station.
OpenExtracts: Extracts have successfully adjusted according to the configuration.
```

**Comment by HB53** [comment]
Just would like to add that I managed to test the Reserve cliff extract and it worked fine :)

**Comment by refringe** [comment]
This should be resolved in [release v1.4.1](https://github.com/refringe/OpenExtracts/releases/tag/v1.4.1). Thank you!
